### ISSUE TITLE
Oakland Housing Authority Application Questions 

### ISSUE THREAD

**Issue by kathyccheng** [bloom-housing/bloom#1199]
As a developer, 
I want applicants to know that they can also apply to Housing Authority preferences on the same property 
So that they can maximize their chances of getting housing 

Implementation Notes 
* Add a screen after the Alameda Live Work preference specifically for Oakland Housing Authority 
* The Oakland Housing Authority "preferences" should show up in the Partners Portal table and CSV export. It could either be a column for each OHA preference + don't consider me,  or a column for OHA and the options are (each preference, None, Don't Consider Me) 
* The Oakland Housing Authority Preferences should NOT show up on the listings page (we'll need to add something custom to hide it just in this instance)
* The title, subtext, and select text are all different here 
* We can use our current preferences model 
* We need a flag for the none of these preferences apply to me 

![image](https://user-images.githubusercontent.com/3453407/116940876-9bfd0e00-ac3c-11eb-91c1-ef1a2499383b.png)

**Comment by seanmalbert** [comment]
From conversation during planning we're split on how to resolve this. @jaredcwhite and I seem to be in agreement that this should be a one off for Alameda for the time being, because we don't have enough information to build a core solution that all clients will benefit from. There is the recognition that the backend core would need an update to even support this. And there is the idea that adding categories to preferences, so we can use those on the frontend to build out where these preferences should appear.

I'm not opposed to a core solution, as long as we can come to an agreement on a solution that is outside of this specific case. @pbn4 has some ideas on how to handle this, and I think if we can get that in writing, we can review these approaches.

**Comment by seanmalbert** [comment]
After talking with @emilyjablonski , here's an update to how we're going to handle this. Per @pbn4 's recommendation, we can handle this with preferences as they currently are with some small adjustments:

1. Add a column to preferences we can use to indicate which preferences should show. If we were to go with something like "page": integer, and have it default to 0 or 1 for the preferences that should show up on the current preferences page, and for each iteration of "page" we display a new set of preferences. This makes it more flexible for future implementation and for this case we don't necessarily have to implement the "pages" aspect in the UI for the first pass. You can just check if it's page 1 or 2.
2. Update sites/public/pages/applications/preferences/all.tsx to display the preferences according to the page they're on. Again for this first pass, if it's relatively much easier to just map them based off of page === [int] then let's go with that, and we can enhance it better in core at a later date.
3. You can get all the preferences in one go and then structure the data and add a useState hook to keep track of which set of preferences you're on. You can up date the onClick handler for the "next" button to move through those preferences, and when you're on the last set, the onClick should take the user through to "Review".

The next part is to handle the clearing of checkboxes when certain checkboxes are clicked. This can be split out into a separate issue that is blocked by this one, if it makes sense to the team.
1. Update preference options (form metadata options) to have a flag which let's us know that if this option is clicked, all other options should be cleared
2. update the onChange handler for the checkbox field so that if that flag is true, the other options get cleared. You may want to add a hook to keep track of which boxes are checked so can easily uncheck them, if that logic isn't already ready available in all.tsx
